components/home/header/Header.tsx

```diff
@@ -22,8 +24,28 @@ export const Header = ({ tagline }: HeaderProps) => (
               Get Started
             </Button>
           </Link>
+          <p className="mt-4">
+            <small>
+              <em>
+                * Sorry,{" "}
+                <Link
+                  href={`/integrations/destinations/${pluginName.toLowerCase()}`}
+                >
+                  <a suppressHydrationWarning>{pluginName}</a>
+                </Link>{" "}
+                - we don't mean to pick on you specifically. We integrate with{" "}
+                <Link href="/integrations">
+                  <a>lots of marketing, operations, and sales tools</a>
+                </Link>
+                .
+              </em>
+            </small>
+          </p>
         </Col>
-        <Col md="6" className={`${styles.image}`}>
+        <Col
+          lg="6"
+          className={`${styles.image} d-md-none d-lg-block text-center`}
```

Comment:
hiding the image on `md`/tablet size because the spacing didn't look right with the first column being longer. Also realized it wasn't centered.